docs/general/fixtures.md

```diff
@@ -0,0 +1,255 @@
+# Fixtures
+
+## Intro
+
+NOTE: This is a living document, renames still have to be done.
+
+You could obtain `SIA_TOKEN` from your `Local Storage` after you logged in into `SIA` with your `ADW account`.
+
+Set `$SIA_TOKEN`:
+
+    SIA_TOKEN=YOUR_SIA_TOKEN_FROM_LOCAL_STORAGE:
+
+You could also use the Chrome console to copy the `accessToken`:
+
+    copy(localStorage.getItem('accessToken'))
+
+Backend OpenAPI documentation:
+
+    https://acc.api.data.amsterdam.nl/api/swagger/?url=/signals/swagger/openapi.yaml#/
+
+## Proposals
+
+### 'Consistent' import naming
+
+This is kind of hard with those long file names...
+
+import privateRolesFixture from 'get-signals-v1-private-roles.json'
+import inputRolesSelectorFixture form  selector-models-categories-selectors_inputRolesSelector.json
+
+## Notes
+
+Tests to fix:
+
+  src/signals/settings/departments/Detail/components/CategoryLists/__tests__/CategoryLists.test.js
+  src/signals/settings/departments/Detail/components/__tests__/mapCategories.test.js
+
+Commands:
```

Comment:
Not sure what the commands below are for.

Reply:
That's why they are in the Notes section, they were useful for me when creating this document, I could remove them or document them, or leave it as is, which one do you prefer?